### PR TITLE
Leave message

### LINKED ISSUE #417: Leaving Message
**Is your feature request related to a problem? Please describe.**

No it's not really related to any problem.

**Describe the solution you'd like**

So When you leave an arena it shows in chat that You left the game.
Example: 

**Describe alternatives you've considered**

This should be added so it would look much better.

**Additional context**

### PR DESCRIPTION
**Pull request by reussy**
Implements a leave message for the player
Closes #417

### CODE REVIEW

File: bedwars-plugin/src/main/java/com/andrei1058/bedwars/language/English.java [hunk 1]
```diff
@@ -111,6 +111,7 @@ public English() {
         yml.addDefault(Messages.COMMAND_COOLDOWN, "&cYou can't do that yet! Wait {seconds} more seconds!");
         yml.addDefault(Messages.ARENA_JOIN_VIP_KICK, "{prefix}&cSorry, but you were kicked out because a donor joined the arena.\n&aPlease consider donating for more features. &7&o(click)");
         yml.addDefault(Messages.ARENA_START_COUNTDOWN_STOPPED_INSUFF_PLAYERS_CHAT, "{prefix}§cThere aren't enough players! Countdown stopped!");
+        yml.addDefault(Messages.PLAYER_LEAVE_ARENA, "{prefix}§cYou're leave of the game!");
```
Comment (ajgeiss0702) [inline bedwars-plugin/src/main/java/com/andrei1058/bedwars/language/English.java]: This should probably be "You left the game!"

"You're leave of the game!" Doesn't really make sense

Comment (reussy) [reply bedwars-plugin/src/main/java/com/andrei1058/bedwars/language/English.java]: Done!

**Comment by andrei1058** [comment]
why?...

**Comment by Memexurer** [comment]
why not :trollface:

**Comment by reussy** [comment]
> why?...

People who use bedwars suggest it, why not?

**Pull request closed without merging**